vulnerable_people_form/form_pages/shared/routing.py

```diff
@@ -160,10 +161,11 @@ def _get_next_form_url_based_on_location_tier(redirect_url, should_redirect_to_n
 def _get_next_form_url_after_shopping_and_priority_supermarket():
     if _is_tiering_logic_enabled():
         location_tier = get_location_tier()
+        shielding_advice = get_shielding_advice()
 
         _validate_location_tier_is_at_least_very_high(location_tier)
 
-        if location_tier == PostcodeTier.VERY_HIGH_PLUS_SHIELDING.value:
+        if shielding_advice == 1:
```

Comment:
is it worth creating an enum for this? As the 1 doesn't really mean anything who is unfamiliar with the code base

Reply:
Sure.